python/nammu/SOAPClient/SOAPClient.py

```diff
@@ -33,13 +37,35 @@ def send(self):
         """
         Elaborate HTTP POST request and send it to ORACC's server.
         """
-        pass
+        url = "http://oracc.museum.upenn.edu:8085"
+        headers = dict(self.request.get_headers())
+        body = self.request.get_body()
+        self.response = requests.post(url, data=body, headers=headers)
+        # try:
```

Comment:
I have to better capture the exceptions from requests. I was working on that before making the changes for this PR, so I'll remove it for now, but will be in the next PR.